webapp/src/components/AssessmentMetadataBar.tsx

```diff
@@ -0,0 +1,260 @@
+        openedTime.getTime() + assessment?.testDuration! * 60 * 1000 <
+          new Date().getTime()
+      ) {
+        setSecondsRemaining(0);
+      } else if (
+        new Date().getTime() + assessment?.testDuration! * 60 * 1000 >
+        dueTime.getTime()
+      ) {
+        setSecondsRemaining(
+          Math.round((dueTime.getTime() - new Date().getTime()) / 1000)
+        );
+      } else {
+        setSecondsRemaining(
+          Math.round(
+            (openedTime.getTime() +
+              assessment?.testDuration! * 60 * 1000 -
+              new Date().getTime()) /
+              1000
+          )
+        );
+      }
+    }
+    previousTimeRef.current = time;
+    requestRef.current = requestAnimationFrame(animate);
+  };
+
+  useEffect(() => {
+    requestRef.current = requestAnimationFrame(animate);
+    return () => {
+      if (requestRef.current) cancelAnimationFrame(requestRef.current);
+    };
+  });
+
+  const [showTimer, setShowTimer] = React.useState(true);
+
+  const handleSetShowTimer = (showTimer: boolean) => () => {
+    setShowTimer(showTimer);
+  };
+
+  return (
+    <>
+      <List
+        sx={{
+          width: '100%',
+          maxWidth: 360,
+        }}
+      >
+        <ListItem>
+          <ListItemAvatar>
+            <Avatar
+              sx={{
+                bgcolor: `${
+                  submission.state === SubmissionStatus.Opened
+                    ? blue[600]
+                    : grey[400]
+                }`,
+              }}
+            >
+              <InfoIcon />
+            </Avatar>
+          </ListItemAvatar>
+          <ListItemText>
+            <Typography variant="body2">{assessment?.description}</Typography>
+            <ListItemText secondary={`Type: ${assessment?.type}`} />
+          </ListItemText>
+        </ListItem>
+        <Divider variant="middle" />
+        <ListItem>
+          <ListItemAvatar>
+            <Avatar
+              sx={{
+                bgcolor: `${
+                  submission.state === SubmissionStatus.Opened
+                    ? blue[600]
+                    : grey[400]
+                }`,
+              }}
+            >
+              <LightbulbIcon />
+            </Avatar>
+          </ListItemAvatar>
+          <ListItemText
+            primary={`${
+              submission?.state === SubmissionStatus.Opened
+                ? `Active`
+                : `${submission?.state}`
+            }`}
+            secondary="Status"
+          />
+        </ListItem>
+        {submission.state === SubmissionStatus.Graded && (
+          <>
+            <ListItem>
+              <ListItemAvatar>
+                <Avatar>
+                  <CheckCircleIcon />
+                </Avatar>
+              </ListItemAvatar>
+              <ListItemText secondary="Score" primary={submission.score} />
+            </ListItem>
+          </>
+        )}
+        {submission.state === SubmissionStatus.Opened && (
+          <ListItem>
```

Comment:
Upon closer thought, this information isn't part of the data being sent back and forth and would be... challenging to implement. So let's drop this section for "Opened" state being a count of number of submissions and file it away in the "nice to have, but not in version 1" bucket.